versioned_docs/version-2.9/how-to-guides/new-user-guides/helm-charts-in-rancher/oci-registries.md

```diff
@@ -0,0 +1,96 @@
+Rancher supports BasicAuth for OCI registries. You must create a [**BasicAuth** Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/#basic-authentication-secret). You can also [create the secret through the Rancher UI](../kubernetes-resources-setup/secrets.md). 
+
+Rancher 2.9.0 also adds the `spec.insecurePlainHttp` field, which allows insecure connections to OCI registries. When this field is set to `true`, Rancher connects to the OCI endpoint without performing an SSL check. This works exactly the same as how the `spec.insecurePlainHttp` field works in [ORAS CLI](https://oras.land/docs/commands/use_oras_cli), since Rancher uses the ORAS library.
+
+The CRD that is linked to the OCI Helm registry is `ClusterRepo`.
+
+[Screenshot of the ClusterRepo YAML]<!-- Engineers - Can we get this screenshot? Thank you! -->
+
+## View Helm Charts in OCI Registries
+
+To view Helm charts in the OCI registry after it achieves an `Active` state:
+
+1. Click **☰**. Under **Explore Cluster** in the left navigation menu, select a cluster.
+1. Click **Apps > Charts**.
+1. Select the OCI repository from the dropdown.
+
+## Refresh an OCI Registry
+
+Rancher automatically refreshes the OCI registry every 6 hours. 
+
+If you need to update immediately, you can perform a manual refresh:
+
+1. Click **☰ > Cluster Management**.
+1. Find the name of the cluster whose repositories you want to access. Click **Explore** at the end of the cluster's row.
+1. In the left navigation bar, select **Apps > Repositories**.
+1. Select the row associated with the OCI registry, and click **Refresh**.
+
+## Update OCI Registry Configuration
+
+1. Click **☰ > Cluster Management**.
+1. Find the name of the cluster whose repositories you want to access. Click **Explore** at the end of the cluster's row.
+1. In the left navigation bar, select **Apps > Repositories**.
+1. Find the row associated with the OCI registry, and click **⋮**.
+1. From the submenu, select **Edit Config**.
+
+## Delete an OCI Registry
+
+1. Click **☰ > Cluster Management**.
+1. Find the name of the cluster whose repositories you want to access. Click **Explore** at the end of the cluster's row.
+1. In the left navigation bar, select **Apps > Repositories**.
+1. Select the row associated with the OCI registry, and click **Delete**.
+
+## Limitations of OCI-based Helm Chart Registries in Rancher
+
+Due to security concerns, there are limitations on how large of a Helm chart you can deploy through an OCI-based registry, and how much metadata you can use to describe the Helm charts within a single OCI endpoint.
+
+Rancher can deploy OCI Helm charts up to 20 MB in size.
+
+The metadata field in index.yaml can store up to 30 MB of information about the Helm charts within an OCI-based registry.
+
+## Troubleshooting OCI-based Helm Registries <!-- Unedited draft -->
+
+To view logs, enable the debug option of rancher 
+
+The first option if there is any discrepancy is to refresh a cluster repository 
```

Comment:
https://github.com/rancher/rancher-docs/pull/1171#discussion_r1534122129

> I left this section unedited because it needs more descriptive instructions. I wasn't sure where in the UI to turn on debugging or which page in the docset we could point to for instructions.